package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "nodemon server.js"
```

Comment:
Make sure to run node when on EC2, I think nodemon is a little less performant than node.